osgp/protocol-adapter-dlms/osgp-protocol-simulator-dlms/simulator/dlms-device-simulator/src/main/java/org/opensmartgridplatform/simulator/protocol/dlms/cosem/PowerQualityExtendedEventLog.java

```diff
@@ -102,7 +102,7 @@ public PowerQualityExtendedEventLog(final Calendar time) {
 
     this.buffer = DataObject.newNullData();
     this.captureObjects = DataObject.newNullData();
-    this.capturePeriod = DataObject.newInteger32Data(CAPTURE_PERIOD);
+    this.capturePeriod = DataObject.newUInteger32Data(CAPTURE_PERIOD);
```

Comment:
This change is just fine, but a few lines down I see: 
```java
this.profileEntries = DataObject.newUInteger16Data(PROFILE_ENTRIES);
```
The method `newUInteger16Data` gives data of type `LONG_UNSIGNED` instead of the expected `DOUBLE_LONG_UNSIGNED`.

Reply:
Changed `newUInteger16Data` in `newUInteger32Data` (also in AuxiliaryEventLog.java)